### DISCUSSION TITLE
How to deploy search-heads without connection to indexers?

### DISCUSSION THREAD

**Discussion by wzhiana** [splunk/splunk-operator#1227] [Q&A] [answered]
Hello, 

a bit wierd request, but for debug some issues we'd like to install search heads without connection to indexers. 

we don't want to deploy it in different namespace or create new cluster. Just remove connection between searchheads and indexers.

**Comment by wzhiana** [comment]
i tried to supply with "fake" data, but obviosly it is not working, as it is not staring containers.

**Reply by akondur** [reply]
Hey @yaroslav-nakonechnikov , can you try deploying without clusterManagerRef on the searchHeadCluster CR?

**Reply by wzhiana** [reply]
i tried, but it was failing to start. 
it was waiting for that field

**Reply by akondur** [reply]
Are you able to open a github issue with details for this?

**Reply by wzhiana** [reply]
but if this is not issue, why i need to report it as a bug? 
question - is to get that info, which is not described in documentation. 

and for now - it is not known - if it is a bug, or a feature. As there is a possibility that it is not possible to deploy part of cluster without connecting it to cluster manager

**Reply by akondur** [reply, marked as answer]
The SearchHeadCluster CRD is independent of the CM, IDXC CRDs(although it has the ability to connect to them via `clusterManagerRef`). If more information is needed, please open a github issue(even though it may not be a bug).
